test/rustfmt/rustfmt_failure_test.sh

```diff
@@ -62,8 +52,20 @@ load("@rules_rust//rust:repositories.bzl", "rust_repositories")
 rust_repositories()
 EOF
 
+  # Drop the 'norustfmt' tags
+  if [ "$(uname)" == "Darwin" ]; then
+    sed -i '' -e 's/"norustfmt"//' "${new_workspace}/test/rustfmt/BUILD.bazel"
```

Comment:
Nit: use a variable for sed flags (SEDOPTS?) and `if` to initialize the variable. The actual sed invocation can then be present just once.

Reply:
Updated.